grid/social-media/posts-comments/profile.supr

```diff
@@ -0,0 +1,130 @@
+name = "social-media/posts-comments"
+version = "1.0.0"
+
+"""
+Get comments on a social media post
+"""
+usecase GetPostComments safe {
+  input {
+    """
+    ID
+    Identifier of the post.
+    """
+    id! string!
```

Comment:
Not sure how we are aligned with other profiles here but I'd consider calling the field `post` or `postId` (unless we say `id` in other, relevant profiles)

Reply:
Isn't that redundant? Or does the attribute have a chance of appearing somewhere on its own (in which case the disambiguation would help)?